src/dotnet/commands/dotnet-pack/PackageGenerator.cs

```diff
@@ -58,7 +59,7 @@ public bool BuildPackage(IEnumerable<ProjectContext> contexts, List<DiagnosticMe
 
             var packageOutputPath = Path.Combine(
                 ArtifactPathsCalculator.PackageOutputPath,
-                GetPackageName() + NuGet.Constants.PackageExtension);
+                GetPackageName() + NuGetConstants.PackageExtension);
```

Comment:
It is necessary otherwise an error is thrown that Constants doesn't exist in the namespace Microsoft.NuGet which I believe is brought in by a build task